akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -141,6 +150,11 @@ object Member {
 
   /**
    * Sort members by age, i.e. using [[Member#isOlderThan]].
+   *
+   * Note that it only makes sense to compare with other members of
+   * same data center. To avoid mistakes of comparing members of different
+   * data centers it will throw `IllegalArgumentException` if the
+   * members belong to different data centers.
```

Comment:
Perhaps we should move this comment to an annotation, `@throws[IllegalArgumentException]("members can only be reliably ordered within a data center")`?

Reply:
I have added annotation, but I would like to keep the explanation in the main doc section because in scaladoc the "message" is rather hidden, and in genjavadoc the "message" is lost

Reply:
ah https://github.com/typesafehub/genjavadoc/issues/84 :)